LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/UdpServer.cs

```diff
@@ -1,413 +0,0 @@
-        {
-            var response = new byte[4]
-            {
-                receivedResults.Buffer[0],
-                receivedResults.Buffer[1],
-                receivedResults.Buffer[2],
-                messageType
-            };
-            _ = UdpSendMessageAsync(response, remoteEndpoint.Address.ToString(), remoteEndpoint.Port);
-        }
-
-        private async Task InitCallBack()
-        {
-            ITransportSettings transportSettings = new AmqpTransportSettings(Microsoft.Azure.Devices.Client.TransportType.Amqp_Tcp_Only);
-
-            ITransportSettings[] settings = { transportSettings };
-
-            // if running as Edge module
-            if (this.configuration.RunningAsIoTEdgeModule)
-            {
-                this.ioTHubModuleClient = await ModuleClient.CreateFromEnvironmentAsync(settings);
-
-                Logger.Init(new LoggerConfiguration
-                {
-                    LogLevel = LoggerConfiguration.InitLogLevel(this.configuration.LogLevel),
-                    LogToConsole = this.configuration.LogToConsole,
-                    LogToUdp = this.configuration.LogToUdp,
-                    LogToUdpPort = this.configuration.LogToUdpPort,
-                    LogToUdpAddress = this.configuration.LogToUdpAddress,
-                    GatewayId = this.configuration.GatewayID
-                });
-
-                if (this.configuration.IoTEdgeTimeout > 0)
-                {
-                    this.ioTHubModuleClient.OperationTimeoutInMilliseconds = this.configuration.IoTEdgeTimeout;
-                    Logger.Log($"Changing timeout to {this.ioTHubModuleClient.OperationTimeoutInMilliseconds} ms", LogLevel.Debug);
-                }
-
-                Logger.Log("Getting properties from module twin...", LogLevel.Information);
-
-                var moduleTwin = await this.ioTHubModuleClient.GetTwinAsync();
-                var moduleTwinCollection = moduleTwin.Properties.Desired;
-                try
-                {
-                    this.loRaDeviceAPIService.URL = new Uri(moduleTwinCollection["FacadeServerUrl"].Value);
-                    Logger.LogAlways($"Facade function url: {this.loRaDeviceAPIService.URL}");
-                }
-                catch (ArgumentOutOfRangeException)
-                {
-                    throw new LoRaProcessingException("Module twin 'FacadeServerUrl' property does not exist.", LoRaProcessingErrorCode.InvalidModuleConfiguration);
-                }
-
-                try
-                {
-                    this.loRaDeviceAPIService.SetAuthCode(moduleTwinCollection["FacadeAuthCode"].Value as string);
-                }
-                catch (ArgumentOutOfRangeException)
-                {
-                    throw new LoRaProcessingException("Module twin 'FacadeAuthCode' does not exist.", LoRaProcessingErrorCode.InvalidModuleConfiguration);
```

Comment:
`LoRaprocessingErorrCode.InvalidModuleConfiguration` became unused, should we remove it?

Reply:
Good point, removed